a11y-speech-synthesis.js

```diff
@@ -0,0 +1,47 @@
+/*global jQuery, wp, SpeechSynthesisUtterance, speechSynthesis, console */
+
+(function( a11y, $ ) {
+	'use strict';
+
+	var utterance = null, originalSpeak;
+
+	if ( 'undefined' === typeof SpeechSynthesisUtterance && 'undefined' === typeof speechSynthesis ) {
+		if ( 'undefined' !== typeof console && 'undefined' !== typeof console.warn ) {
+			console.warn( 'Accessibility (a11y) Speech Synthesis not supported by this browser.' );
+		}
+		return;
+	}
+
+	originalSpeak = a11y.speak;
+
+	/**
+	 * Update the ARIA live notification area text node using speechSynthesis.
+	 *
+	 * @since 4.2.0
+	 * @since 4.3.0 Introduced the 'ariaLive' argument.
+	 *
+	 * @param {String} message  The message to be announced by Assistive Technologies.
+	 * @param {String} ariaLive Optional. The politeness level for aria-live. Possible values:
+	 *                          polite or assertive. Default polite.
+	 * @returns {void}
+	 */
+	a11y.speak = function speak( message, ariaLive ) {
+		var messageText;
+
+		// Clear any current utterance.
+		if ( utterance ) {
+			speechSynthesis.cancel( utterance );
```

Comment:
@afercia Is this the correct interpretation of `clear()`? In other words, if I do `wp.a11y.speak( 'foo' ); wp.a11y.speak( 'bar' );` will a screen reader only utter “bar” with “foo” being immediately stopped?

https://github.com/WordPress/wordpress-develop/blob/5af4552f6d87b7d4c85765a032863a83d6915079/src/wp-includes/js/wp-a11y.js#L22-L23